thunder/executors/data_dependent_partition.py

```diff
@@ -103,7 +103,8 @@ def __init__(self, trace: TraceCtx):
                 for copy_node in copy_nodes:
                     node.parents.add(copy_node)
                     copy_node.children.add(node)
-            elif bsym.sym.id is PrimIDs.COPY_:
+            # torch executor replaces prims.copy_ with torchex.copy_
+            elif bsym.sym.id in (PrimIDs.COPY_, "copy_"):
```

Comment:
could you check if `torchex.copy_.id` instead of "copy_" works?

Reply:
Thanks, I changed it to comparison between syms.